react-native-libraries.json

```diff
@@ -373,7 +373,8 @@
     "ios": true,
     "android": true,
     "web": false,
-    "expo": true
+    "expo": true,
+    "windows": true
```

Comment:
Hi @asklar! I think that you have in mind `@react-native-community/async-storage` package, not this one, right?

Reply:
yep good catch, I'll change it :)